### PR TITLE
chore: remove unnecessary dependencies

### PR DESCRIPTION
**Pull request by sidvishnoi**

### CODE REVIEW

File: .eslintrc.json [hunk 1]
```diff
@@ -54,5 +52,6 @@
     "pickRandomsFromList": true,
     "respecConfig": true
   },
+  "ignorePatterns": ["src/core/text-loader.js"],
```
Comment (sidvishnoi) [inline .eslintrc.json:55]: Waiting on https://github.com/eslint/eslint/pull/13196. Can get rest of the PR in before that though, but no hurries.

**Comment by sidvishnoi** [comment]
@marcoscaceres I was thinking of also removing `opencollective` and `opencollective-postinstall`. This just slows down installations and is a bit noisy. I don't think anyone donates by seeing those messages in their CI either.

Also, do we need all the karma browser launchers?

File: tests/headless.js [hunk 2]
```diff
@@ -67,8 +66,9 @@ async function runRespec2html() {
 }
 
 function debug(msg) {
+  const currentTime = new Date().toLocaleTimeString("en-US");
   // eslint-disable-next-line no-console
-  console.log(colors.grey(moment().format("LTS")) + colors.cyan(` ${msg}`));
+  console.log(colors.grey(currentTime) + colors.cyan(` ${msg}`));
```
Comment (marcoscaceres) [inline tests/headless.js:71]:
```suggestion
  console.log(`${colors.grey(currentTime)} ${colors.cyan(msg)`);
```

File: package.json [hunk 3]
```diff
@@ -21,43 +21,33 @@
...
     "hyperhtml": "^2.32.2",
     "idb": "^5.0.1",
-    "jasmine-core": "^3.5.0",
-    "jasmine-reporters": "^2.3.2",
+    "jasmine": "^3.5.0",
     "jquery": "^3.4.1",
     "karma": "^4.4.1",
     "karma-chrome-launcher": "^3.1.0",
-    "karma-detect-browsers": "^2.3.3",
     "karma-edge-launcher": "^0.4.2",
```
Comment (marcoscaceres) [inline package.json:42]: this can probably go too I guess... it's just Chrome.

File: package.json [hunk 4]
```diff
@@ -21,43 +21,33 @@
...
     "chai": "^4.2.0",
     "chokidar": "^3.3.1",
     "clean-css": "^4.2.3",
-    "domReady": "github:requirejs/domReady#d85bdc38a6df868ead52ab3e80715aaf60e68765",
     "eslint": "^6.8.0",
     "eslint-config-prettier": "^6.10.1",
     "eslint-plugin-import": "^2.20.1",
     "eslint-plugin-jasmine": "^4.1.0",
     "eslint-plugin-prettier": "^3.1.2",
-    "handlebars": "^4.7.3",
```
Comment (marcoscaceres) [inline package.json:38]: We might need to bring this back if the Geonovum folks finish migrating over their profile... they are still using it, I think.

File: package.json [hunk 5]
```diff
@@ -21,43 +21,33 @@
...
-    "karma-detect-browsers": "^2.3.3",
     "karma-edge-launcher": "^0.4.2",
     "karma-firefox-launcher": "^1.3.0",
     "karma-jasmine": "^3.1.1",
     "karma-jasmine-html-reporter": "^1.5.3",
     "karma-mocha": "^1.3.0",
     "karma-mocha-reporter": "^2.2.5",
-    "karma-opera-launcher": "^1.0.0",
     "karma-safari-launcher": "^1.0.0",
     "karma-safaritechpreview-launcher": "2.0.2",
```
Comment (marcoscaceres) [inline package.json:49]: Can probably get rid of this one too...

File: package.json [hunk 6]
```diff
@@ -87,19 +77,16 @@
...
     "test:headless": "node ./tests/headless.js",
-    "test:karma": "npm run karma",
-    "postinstall": "opencollective-postinstall"
+    "test:karma": "npm run karma"
   },
   "dependencies": {
     "colors": "^1.4.0",
     "command-line-args": "^5.1.1",
     "command-line-usage": "^6.1.0",
     "epipebomb": "^1.0.0",
```
Comment (marcoscaceres) [inline package.json:88]: We might not need this anymore in newer node:
https://github.com/mhart/epipebomb/issues/10

File: package.json (hunk 3 above)
Comment (sidvishnoi) [reply package.json:42]: @saschanaz might be using this?

Comment (saschanaz) [reply package.json:42]: Nope 😁

File: package.json (hunk 4 above)
Comment (sidvishnoi) [reply package.json:38]: @thijsbrentjens Would you be needing handlebars?

File: package.json (hunk 6 above)
Comment (sidvishnoi) [reply package.json:88]: We're supporting Node 12 presently though, as 14 isn't LTS yet.

Comment (marcoscaceres) [reply package.json:86]: ah... true! Booooo!!!!

File: package.json (hunk 4 above)
Comment (marcoscaceres) [reply package.json:38]: Oh, maybe not... all the templates are using hyperHTML! https://github.com/Geonovum/respec/tree/develop/src/geonovum/templates

Comment (marcoscaceres) [reply package.json:38]: As an aside, looks like downstream is getting really out of date... @thijsbrentjens, we should look at moving more stuff over, as Geonovum might be missing out on important features and security updates.

Comment (sidvishnoi) [reply package.json:38]: Thanks for confirming!

**Pull request merged**